modules/calendar/event.py

```diff
@@ -24,8 +24,8 @@ def __init__(
 		self.__location = location
 		self.__description = description
 		self.__all_day = all_day
-		self.__start = start
-		self.__end = end
+		self.__start = start.replace(tzinfo=None)
+		self.__end = end.replace(tzinfo=None)
 		self.__timezone = timezone
```

Comment:
does event still need a timezone if youre specifically removing the timezone?

Reply:
It's only removing the timezone from the datetime objects so you can compare with other date objects, but the timezone property is still relevant.

I plan on fixing issues with different timezones soon.